client/login/wp-login/login-links.jsx

```diff
@@ -237,6 +239,17 @@ export class LoginLinks extends React.Component {
 			signupUrl += '/' + signupFlow;
 		}
 
+		if ( config.isEnabled( 'signup/wpcc' ) && isCrowdsignalOAuth2Client( oauth2Client ) ) {
+			const oauth2Flow = 'crowdsignal';
+			const redirectTo = get( currentQuery, 'redirect_to', '');
```

Comment:
@ice9js Wondering if this needs to be made more specific, i.e. not accept [open redirects](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.md) or if this is handled elsewhere in the Calypso code

Reply:
It's handled by a check that validates `redirect_to` against the given `oauth2_client_id` when the page is loaded so it should be fine.

Reply:
Got it! You mean the whole url or the id?

Reply:
Specifically, it checks if the url from `redirect_to` matches the domain that's specified for the current OAuth2 client, if that's what you're asking.

Reply:
Thanks, got me covered! :) 💯